lib/renderer/api/context-bridge.ts

```diff
@@ -1,13 +1,21 @@
 const binding = process._linkedBinding('electron_renderer_context_bridge');
 
 const checkContextIsolationEnabled = () => {
-  if (!process.contextIsolated) throw new Error('contextBridge API can only be used when contextIsolation is enabled');
+  if (!process.contextIsolated) {
```

Comment:
It looks like you've run a code formatter on our files that _isn't_ our own linting rules, can you undo these changes to make this diff easier to review.